extension/src/vscode/extensions.ts

```diff
@@ -26,16 +26,6 @@ export const getExtensionAPI = async <T>(
   } catch {}
 }
 
-export const getExtensionVersion = <T>(id: string): string | undefined => {
```

Comment:
Is this an information that could be useful? Maybe this could be added in the `sendTelemetryEvent` function just like the user_id.

Reply:
it gets automatically sent by the Telemetry now, it picks up the information itself